.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      actions:
+        patterns:
+          - "*"
+    labels:
+      - "github-actions"
+      - "dependencies"
```

Comment:
This will automatically apply these labels to the PRs as well. If these labels don't exist yet it will complain for people to make them, but won't fail.